Stripe/UIViewController+Stripe_Alerts.m

```diff
@@ -15,47 +15,6 @@
 
```

Comment:
I think this file can probably be deleted outright now, as it no longer really has any advantages over just using a bare `UIAlertController`.